scripts/tools/common.yml

```diff
@@ -157,7 +169,7 @@ etors:
     - name: UINT32
       desc: "32-bit unsigned-integer"
     - name: UINT64
-      desc: "64-bit unsigned-integer"
+      desc: "32-bit unsigned-integer"
```

Comment:
32 -> 64

Reply:
This was fixed in this PR earlier.
But received a comment to revert, since it may not be related to this scope.
Reference : https://github.com/oneapi-src/level-zero-spec/pull/204#discussion_r1310855530

However, I can still go ahead and fix this. Please suggest.

Reply:
@wdamon-intel 
fixed it.